nasa/harmony-regression-tests
# Das 1867

## Description

This PR just removes the nifty `raises-exception` tag from the notebook to make sure that failures are actually caught.

## Jira Issue ID
[DAS-1867|https://bugs.earthdata.nasa.gov/browse/DAS-1867]

## Local Test Steps

1. Run the success cases.

Pull this branch and execute the `TrajectorySubsetter_Regression.ipynb` notebook. Each test should pass.

2. Run failure cases.

Edit the arguments for `compare_results_to_reference_file()` in each test to reference the wrong file. Each of these tests should fail.

## PR Acceptance Checklist
